generators/client/templates/react/src/test/javascript/spec/app/shared/reducers/user-management.spec.ts.ejs

```diff
@@ -1,99 +0,0 @@
-<%#
```

Comment:
This file wasn't copied into the client, there is another user-management.spec.ts file [here](https://github.com/jhipster/generator-jhipster/blob/f1c05547f556c371adb3832fc2e7ed0bdf5b4919/generators/client/templates/react/src/test/javascript/spec/app/modules/administration/user-management/user-management.reducer.spec.ts.ejs)

Reply:
I may be wrong, but, think this was for OIDC flow.

Reply:
Can't believe I missed that